specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/preview/2021-07-01-preview/virtualNetworks.json

```diff
@@ -0,0 +1,588 @@
+          "type": "string",
+          "description": "Type of the network",
+          "enum": [
+            "NAT",
+            "Transparent",
+            "L2Bridge",
+            "L2Tunnel",
+            "ICS",
+            "Private",
+            "Overlay",
+            "Internal",
+            "Mirrored"
+          ],
+          "x-ms-enum": {
+            "name": "NetworkTypeEnum",
+            "modelAsString": true
+          }
+        },
+        "subnets": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "name": {
+                "type": "string",
+                "description": "Name - The name of the resource that is unique within a resource group. This name can be used to access the resource."
+              },
+              "properties": {
+                "type": "object",
+                "x-ms-client-flatten": true,
+                "properties": {
+                  "addressPrefix": {
+                    "type": "string",
+                    "description": "Cidr for this subnet - IPv4, IPv6"
+                  },
+                  "addressPrefixes": {
+                    "type": "array",
+                    "items": {
+                      "type": "string"
+                    },
+                    "description": "AddressPrefixes - List of address prefixes for the subnet."
+                  },
+                  "ipAllocationMethod": {
+                    "type": "string",
+                    "description": "IPAllocationMethod - The IP address allocation method. Possible values include: 'Static', 'Dynamic'",
+                    "enum": [
+                      "Dynamic",
+                      "Static"
+                    ],
+                    "x-ms-enum": {
+                      "name": "IpAllocationMethodEnum",
+                      "modelAsString": true
+                    }
+                  },
+                  "ipConfigurationReferences": {
+                    "type": "array",
+                    "items": {
+                      "type": "object",
+                      "properties": {
+                        "ID": {
+                          "type": "string",
+                          "description": "Name of the Ip Configuration"
+                        }
+                      },
+                      "description": "IPConfigurationReference - Describes a IPConfiguration under the virtual network"
+                    },
+                    "x-ms-identifiers": [],
+                    "description": "IPConfigurationReferences - list of IPConfigurationReferences"
+                  },
+                  "routeTable": {
+                    "type": "object",
+                    "properties": {
+                      "id": {
+                        "type": "string",
+                        "description": "Etag - Gets a unique read-only string that changes whenever the resource is updated."
```

Comment:
I am not sure the description is correct here. Thoughts @Arkanayan ?

Reply:
Yeah, the description is incorrect. I have updated it to "Name of the Route". How does it sound? I have never used this field.

Reply:
let's call it "route ID - ID of the IP route"

Reply:
Resolved.